reconstruct/reconstruct.go

```diff
@@ -3,39 +3,73 @@ package reconstruct
 import (
 	"github.com/vertgenlab/gonomics/dna"
 	"github.com/vertgenlab/gonomics/expandedTree"
+	"github.com/vertgenlab/gonomics/fasta"
 	"log"
 )
 
-//test accuracy of the reconstruction compared to the simulation
-//func Accuracy(simFilename string, recFilename string) float64 {
-//	tot := 0.0                     //why is this here when it could just be num?
-//	sim := fasta.Read(simFilename) //produced by simulate.printSeqForNodes, returned by simulate.Simulate
-//	rec := fasta.Read(recFilename)
-//	des := "descendents_" + simFilename //refers to file created by simulate.RemoveAncestors which has only leaf nodes labelled
-//	simLeaves := fasta.Read(des)
-//	for i := 0; i < len(sim); i++ {
-//		for j := 0; j < len(simLeaves); j++ {
-//			if sim[i].Name == simLeaves[j].Name { //if current fasta is a leaf fasta
-//				sim = append(sim[:i], sim[i+1:]...)
-//				//sim is fastas of every node but the leaf nodes, so we are adding leaf fastas to the original simulation file
-//			}
-//		}
-//	}
-//	for i := 0; i < len(sim); i++ {
-//		num := 0.0
-//
-//		for k := 0; k < len(sim[0].Seq); k++ {
-//			if sim[i].Seq[k] != rec[i].Seq[k] {
-//				num = num + 1
-//			}
-//		}
-//		tot = tot + num
-//	}
-//	accuracy := tot / (float64(len(sim)) * float64(len(sim[0].Seq))) * 100.0
-//	acc := 100 - accuracy
-//	fmt.Print("accuracy over all nodes= ", acc, "%", "\n")
-//	return acc
-//}
+//returns the percentage accuracy by base returned by reconstruct of each node and of all nodes combined (usage in reconstruct_test.go)
+func ReconAccuracy(simFilename string, reconFilename string) map[string]float64 {
```

Comment:
This looks like the same/similar function to the one in the command.  I think you will want to remove the one in the command (see if my comments from that one apply to this one) and then in the command you would just call this function and then loop through the map you get back to print the accuracies (you already have that loop at the end of that function).